napari/_qt/widgets/qt_dims_slider.py

```diff
@@ -642,18 +600,20 @@ def work(self):
                 self.frame_requested.emit(self.axis, self.min_point)
             elif self.step < 0 and self.current <= self.min_point + 1:
                 self.frame_requested.emit(self.axis, self.max_point)
-            self.timer.start()
+            # self.timer.start()
         else:
             # immediately advance one frame
             self.advance()
-        self.started.emit()
+        # next_time = time.time() + steep
```

Comment:
```suggestion
        # next_time = time.time() + step
```

But maybe remove the comment?

Reply:
removed